src/components/color-modes.tsx

```diff
@@ -7,60 +7,80 @@ import { Theme } from '@blockstack/ui';
 
 export const colorGet = (path: string, fallback?: string) => themeGet('colors.' + path, fallback);
 
-interface colorModeTypes {
-  ['accent']: string;
-  ['bg']: string;
-  ['bg-alt']: string;
-  ['bg-light']: string;
-  ['invert']: string;
-  ['text-hover']: string;
-  ['text-title']: string;
-  ['text-caption']: string;
-  ['text-body']: string;
-  ['input-placeholder']: string;
-  ['border']: string;
-  ['feedback-alert']: string;
-  ['feedback-error']: string;
-  ['feedback-success']: string;
+enum Color {
```

Comment:
Love this file. It would be awesome if we could update UI with dark/light themes by default.

Reply:
Would love that too, but it'll be somewhat of a large undertaking to convert everything over.